services/frontend-service/src/ui/utils/store.tsx

```diff
@@ -487,7 +487,9 @@ export const useCurrentlyDeployedAtGroup = (application: string, version: number
 // Get release information for a version
 export const useReleaseInfo = (app: string, version: number): Release =>
     useOverview(({ applications }) => {
-        const releaseInfo = applications[app]?.releases.filter((release) => release.version === version)[0];
+        const releaseInfo = applications[app]?.releases.filter((r) =>
```

Comment:
if you just want the first result, use `find` instead of `filter`.

Reply:
I will leave this as it is too.